agent/engine/docker_container_engine.go

```diff
@@ -146,9 +147,13 @@ func NewDockerGoClient(clientFactory dockerclient.Factory, cfg *config.Config) (
 		return nil, err
 	}
 
+	dockerAuthData := json.RawMessage("")
```

Comment:
Can you use `var dockerAuthData json.RawMessage` instead? Somehow seems cleaner to me than parsing an empty string. FWIW, this is how it's being initialized in encoding/stream.go: https://golang.org/src/encoding/json/stream.go#L266